CONTRIBUTING.md

```diff
@@ -889,3 +893,6 @@ end
 ```
 
 [Back to code style TOC](#table-of-contents---code-style)
+[Back to TOC](#table-of-contents)
```

Comment:
This one already has a back link next to it — could you remove this one?

Reply:
sure